docs/support/account.md

```diff
@@ -47,35 +54,37 @@ ii. After completing this, reset MFA for [WOG account](reset-security-verificati
+When public officers sign up for TechPass, a TechPass account is created but will be in pending status until they accept the invitation link. If they do not accept this invitation link within 30 days, this account will be deleted. Note this is different from disabling TechPass account. If you still want a TechPass account, from your GSIB device, go to [TechPass portal](http://portal.techpass.gov.sg) and sign up again to receive a new invitation link.
 
 </details>
 
 <details><summary>7. Why is my TechPass account disabled? How to re-enable it?</summary>
 
 Your TechPass account might be disabled if you have not used it for 90 consecutive days. However, if you have not used it for 60 consecutive days, from day 61 onwards you will receive an email alert about your inactive status with the remediation step. If you still do not use your TechPass account, your account will be disabled on day 90 and you will be notified.
 
-To re-enable or if you think your account was incorrectly disabled, raise a [service request](https://go.gov.sg/techpass-sr) with us.
+To re-enable or if you think your account was incorrectly disabled, create a [service request](https://go.gov.sg/techpass-sr).
 
 </details>
 
 <details><summary>8. I am a public officer and unable to sign in to my WOG account from my GMD.</summary>
 
-You might encounter this error if you are trying to sign in to your WOG account without setting up the MFA to authenticate it. For more information, refer to [step 3 in Onboarding public officer](https://docs.developer.tech.gov.sg/docs/techpass-user-guide/#/onboard-public-officers-using-non-se-machines?id=step-3-set-up-security-verification-for-your-wog-account)
+![mfa_error](../assets/support/mfa_error.jpg)
+
+You might encounter this error if you are trying to sign in to your WOG account without setting up the MFA to authenticate it. For more information, refer to [step 1 in Onboarding public officer](https://docs.developer.tech.gov.sg/docs/techpass-user-guide/#/onboard-public-officers-using-non-se-machines?id=step-1-set-up-security-verification-for-your-wog-account)
 
 </details>
 
-![mfa_error](../assets/support/mfa_error.jpg)
+
 
 [reset-password]: https://passwordreset.microsoftonline.com/
 [password-policy-of-azure-active-directory]: https://docs.microsoft.com/en-us/azure/active-directory/authentication/concept-sspr-policy#administrator-password-policy-differences
 [reset-password-gsib]: https://itsm.sgnet.gov.sg/sp3
 [service-request]: https://go.gov.sg/techpass-sr
 [reset-password-vendor]: https://passwordreset.microsoftonline.com/
-[reset-mfa]: ../reset-mfa.md
+[reset-mfa]: https://docs.developer.tech.gov.sg/docs/techpass-user-guide/#/reset-mfa
```

Comment:
hmmm.. must this be a absolute url? cos this will not be pointing to the right place in staging docs

Reply:
Updated. Thanks!

Reply:
So is this the best practice: (i would like to update the documentation handbook :-)
Use an absolute path only to link to an external page.
use a relative path to link to pages/sections within the repository.